packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -139,19 +138,24 @@ class ExpansionPanelRadio extends ExpansionPanel {
 /// A material expansion panel list that lays out its children and animates
 /// expansions.
 ///
-/// Note that [expansionCallback] behaves differently for [ExpansionPanelList]
-/// and [ExpansionPanelList.radio].
+/// The [expansionCallback] is called when the expansion state changes. For
+/// normal [ExpansionPanelList] widgets, it is the responsibilty of the parent
+/// widget to rebuild the [ExpansionPanelList] with updated values for
+/// [ExpansionPanel.isExpanded]. For [ExpansionPanelList.radio] widgets, the
+/// callback is invoked both for the previously open panel, which is closing,
+/// and the previously closed panel, which is opening, and the widget tracks the
+/// open state internally.
```

Comment:
The last "and the widget tracks the open state internally" statement feels disjointed to me. Consider reordering:

```suggestion
/// [ExpansionPanel.isExpanded]. For [ExpansionPanelList.radio] widgets, the
/// open state is tracked internally and the callback is invoked 
/// both for the previously open panel, which is closing, and the
// previously closed panel, which is opening.
```

(This suggestion's formatting is likely off)